struct/class declaration mixup in nodelet

```
/Users/mikepurvis/ros-install-osx/lunar_desktop_full_ws/src/nodelet_core/nodelet/include/nodelet/detail/callback_queue_manager.h:87:9: warning: class member cannot be redeclared [-Wredeclared-class-member]
  class ThreadInfo;
        ^
/Users/mikepurvis/ros-install-osx/lunar_desktop_full_ws/src/nodelet_core/nodelet/include/nodelet/detail/callback_queue_manager.h:84:10: note: previous declaration is here
  struct ThreadInfo;
         ^
1 warning generated.
```

--- a/nodelet/include/nodelet/detail/callback_queue_manager.h
+++ b/nodelet/include/nodelet/detail/callback_queue_manager.h
@@ -84,7 +84,6 @@ private:
   struct ThreadInfo;
   void workerThread(ThreadInfo*);
 
-  class ThreadInfo;
   ThreadInfo* getSmallestQueue();
 
   struct QueueInfo

